app/invocation/invocation_execution_card.tsx

```diff
@@ -264,14 +272,22 @@ export default class ExecutionCardComponent extends React.Component {
       }
     }
 
-    const filteredActions = this.state.executions.filter(
-      (action) =>
-        !this.props.filter ||
-        `${action.actionDigest.hash}/${action.actionDigest.sizeBytes}`
-          .toLowerCase()
-          .includes(this.props.filter.toLowerCase()) ||
-        action.commandSnippet.toLowerCase().includes(this.props.filter.toLowerCase())
-    );
+    const filteredActions = this.state.executions
+      .filter(
+        (action) =>
+          !this.props.filter ||
+          `${action.actionDigest.hash}/${action.actionDigest.sizeBytes}`
+            .toLowerCase()
+            .includes(this.props.filter.toLowerCase()) ||
+          action.commandSnippet.toLowerCase().includes(this.props.filter.toLowerCase())
+      )
+      .filter(
+        (action) =>
+          this.state.statusFilter.includes("all") ||
+          getExecutionStatus(action).name.toLowerCase().includes(this.state.statusFilter.toLowerCase()) ||
+          (getExecutionStatus(action).name.toLowerCase().includes("failed") &&
```

Comment:
I think the logic should be: If the lowercase `executionStatus.name` starts with "failed" then we should show it only if `filter === "failed'` -- and if it starts with "Error" then we show it only if `filter === "error"` (we want `startsWith` instead of `includes` bc errored actions might include FAILED, e.g. `Error (FAILED_PRECONDITION)`)

Reply:
Ah I see, I thought we wanted to consolidate the error & failed ones into one option